### DISCUSSION TITLE
Grouping Panels in Grafana

### DISCUSSION THREAD

**Discussion by therealdoodly** [grafana/grafana#72183] [Alerting]
I'm currently developing an interface for machines but there is a problem. I will try to describe it (Dasboard is in German). I'm using the feature Repeat options, because I want to repeat multiple Panels in my Dashboard. The max per row is on 6. But there is one Problem I can't define how they should repeat after the 6th row. They are getting placed in a way I don't like here are 2 pictures which describe it:
![image](https://github.com/grafana/grafana/assets/54245428/f6f3c04e-4a3f-49bb-8025-ea2b6a2fb633)
![image](https://github.com/grafana/grafana/assets/54245428/302409ad-f4e8-442a-ad9f-89774eead37f)
Is there a way to group the panels. I would like to repeat them as a group. Is there a plugin that helps with this problem or a planned or existing feature? Thx for your help

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82625. Feel free to continue the discussion around this there. Thank you!
